src/containerapp/azext_containerapp/tests/latest/test_containerapp_scenario.py

```diff
@@ -281,72 +280,6 @@ def test_containerapp_update_containers(self, resource_group):
             JMESPathCheck('properties.template.containers[1].resources.memory', '1.5Gi'),
         ])
 
-    # TODO fix and enable
-    @unittest.skip("API only on stage currently")
-    @live_only()  # VCR.py can't seem to handle websockets (only --live works)
-    # @ResourceGroupPreparer(location="centraluseuap")
-    @mock.patch("azext_containerapp._ssh_utils._resize_terminal")
-    @mock.patch("sys.stdin")
-    def test_containerapp_ssh(self, resource_group=None, *args):
```

Comment:
why remove this one? is it removed to cli core?

Reply:
This test has been moved to Core.